Formula/boost@1.55.rb

```diff
@@ -17,6 +17,8 @@ class BoostAT155 < Formula
 
   keg_only :versioned_formula
 
+  disable! because: "old versioned formula"
```

Comment:
```suggestion
  disable! because: :versioned_formula
```

We can use symbols now that https://github.com/Homebrew/brew/pull/8530 has been merged (and a new release was tagged)

The text for `:versioned_formula` is `is a versioned formula` but I wouldn't be opposed to changing it to `is an old versioned formula` if that's clearer.